### PR TITLE
Update notebook to use Gemini model instead of text-bison because text-bison will be deprecated.

### PR DESCRIPTION
**Pull request by siping-hu**
**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

### CODE REVIEW

File: notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb [hunk 1]
```diff
@@ -83,7 +83,7 @@
         "This tutorial uses the following Google Cloud ML services and resources:\n",
         "\n",
         "- Cloud Storage\n",
-        "- Vertex AI PaLM API\n",
+        "- Vertex AI Gemini API\n",
         "- Vertex AI Pipelines\n",
         "- Vertex AI Batch Prediction\n",
```
Comment (kittyabs) [inline notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb:88]: - change: "Vertex AI Batch Prediction"
- to: "Vertex AI batch prediction

**Review by kittyabs — COMMENTED** [review]
product name edit

File: notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb (hunk 1 above)
Comment (siping-hu) [reply notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb]: done.

File: notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb [hunk 2]
```diff
@@ -577,7 +579,7 @@
         "id": "veq26QZ7OMoC"
       },
       "source": [
-        "In this notebook, we will evaluate a third-party model's predictions (located in the `summary` column of `DATASET`) against the output of `text-bison@001` using a built-in summarization instruction. The task being performed is summarization.\n",
+        "In this notebook, we will evaluate a third-party model's predictions (located in the `summary` column of `DATASET`) against the output of `gemini-pro 1.0` using a built-in summarization instruction. The task being performed is summarization.\n",
```
Comment (TheMichaelHu) [inline notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb:582]: nit: `gemini-1.0-pro` or Gemini 1.0 Pro

Also, can we use 1.5 Flash or Pro instead? I think we're trying to encourage users to use 1.5

Comment (siping-hu) [reply notebooks/official/model_evaluation/model_based_llm_evaluation/autosxs_llm_evaluation_for_summarization_task.ipynb]: Done. Tested with 1.5 pro on my side.

**Review by gericdong — APPROVED** [review]

**Pull request merged**